src/app/app-routing.module.ts

```diff
@@ -30,7 +31,8 @@ const routes: Routes = [
   {path: 'host', component: HostSessionComponent},
   {path: 'host/:sessionId', component: LargeDisplayComponent},
   {path: 'test', component: TestPagesComponent},
-  {path: 'test', children: testRoutes}
+  {path: 'test', children: testRoutes},
+  {path: 'privacy-policy', component: PrivacyPolicyComponent}
```

Comment:
Joe and I think we should have the privacy policy and TOS be stand-alone static HTML pages rather than Angular components that require loading the whole app to view. This is because users checking out the project on GitHub or other places may want to be able to view the privacy policy and TOS without actually loading the tracking and account code that those policies are about. It is also useful to have them as separate static pages for listing on app stores.